website/templates/profile/notifications.mako

```diff
@@ -44,13 +45,23 @@
                     <!-- Flashed Messages -->
                     <div data-bind="html: message, attr: {class: messageClass}"></div>
             </div><!--view model scope ends -->
-            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><h3 class="panel-title">Configure Notification Preferences</h3></div>
+                <form id="selectNotifications" class="osf-treebeard-minimal">
+                    <div id="grid"></div>
+                    <div class="help-block" style="padding-left: 15px">
+                            <p id="configureNotificationsMessage"></p>
+                    </div>
+                </form>
```

Comment:
I agree with BrianG in that since this is probably something that we want to work toward and I've already implemented it, there's no need to revert back to using a submit button only to then change it back to auto-submit at a later date.